### ISSUE TITLE
Add Dark and Light mode feature

### ISSUE THREAD

**Issue by darkshredder** [CircuitVerse/CircuitVerse#958]
**Is your feature request related to a problem? Please describe.**
Many people like to work at night or any other time and would love to work in their desired mode
**Describe the solution you'd like**
So I would like to create a toggle button which will change the theme to light or dark mode

**Are you working on this? (Yes/No)**
Yes

**Comment by akansha-singh** [comment]
@darkshredder are you still working on this issue ?

**Comment by darkshredder** [comment]
not right now

**Comment by aayush-05** [comment]
@satu0king @tachyons @nitin10s are we planning something like this anytime soon?

**Comment by ryuk156** [comment]
@aayush-05  for this I think  we need to refactor  the css the buttons in the CV and other components  dont have same code for all the buttons  their  should be linear code and also components  that are same

**Comment by aayush-05** [comment]
Yes @ryuk156 , refactoring for frontend codebase is planned to be completed after we get over revamping the left over pages and fixing some workflows. I am just unsure if this a priority atm.

**Comment by ryuk156** [comment]
Okay😄

**Comment by nitin10s** [comment]
@ryuk156 is right. We can look at this as an option in future surely.

**Comment by vedant-jain03** [comment]
@tachyons Can I work on this?

**Comment by nitin10s** [comment]
Lets wait on this issue while we fix all the UI problems on the normal version.

**Comment by vedant-jain03** [comment]
> Lets wait on this issue while we fix all the UI problems on the normal version.

Sure @nitin10s

**Comment by nisha-laur** [comment]
@nitin10s @vedant-jain03 @darkshredder Can I work on this issue ?

**Comment by orbaps** [comment]
Can I work on this

**Comment by sadaaizaz** [comment]
Hi! I’m a new contributor and would love to work on this if it’s available. I’ve set up the repo locally and started exploring the codebase. Please let me know if I can take this up. Thanks!

**Comment by VijayabaskarR-06** [comment]
I’m interested in implementing this feature.
I plan to explore theming support and would appreciate any guidance on architecture or priorities.

**Comment by ChandrakantaMandal** [comment]
Hi maintainers 👋
I’m interested in working on this issue and would love to contribute. If the issue is still open, please assign it to me.
Please let me know if there are any guidelines I should follow before starting.
